staking/stakingprecompile.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Staking Precompile
+description: Moonbeam Parachain Staking Ethereum Solidity Precompile Interface Demo
+---
+
+# Staking Precompile
+
+![Staking Moonbeam Banner](/images/staking/staking-stake-banner.png)
+
+## Moonbeam Parachain-Staking
+
+A delegated proof of stake pallet recently debuted called [Parachain-Staking](https://github.com/PureStake/moonbeam/tree/master/pallets/parachain-staking/src), allowing token holders (nominators) to express exactly which collator candidates they would like to support and with what quantity of stake. The design of the Parachain-Staking pallet is such that it enforces shared risk/reward on chain between delegators and collators.
+
+The Staking module is coded in Rust and it is part of a pallet that is normally not accessible from the Ethereum side of Moonbeam. However, a Staking Precompile allows developers to access the staking features as a precompiled contract located at address 0x0000000000000000000000000000000000000800. The Staking Precompile is included in our [Moonbase Alpha v8 release](https://moonbeam.network/announcements/testnet-upgrade-moonbase-alpha-v8/).
```

Comment:
Remember to use the `variables.yml` file for stuff that are specific and might change in the future. `0x0000000000000000000000000000000000000800` I would do like {{ networks.moonbase.staking.precompile }} or something similar.

Also, make sure to wrap the value in backticks.